module3/labs/Week1/HTMLCSSFundamentals.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+title: HTML and CSS Lab
+---
+
+## Recreating Comps
+
+We are going to be practicing our HTML and CSS skills by re-creating images of web-pages.
+
+## Setup
+* Fork this [starter repl](https://replit.com/@launch-team/staticchallenges#index.html)
+    * The content for the page is in `site_copy.md`
+    * All images are included in the `images` directory
+* Choose one of the Comp Images to recreate from below.
+* Update the `index.html` and `stylesheets/main.css` files to recreate one of the images below.
+    * Use at least 5 html tags
+    * Use at least 2 classes
+    * Use at least 2 ids
+    * make sure to use semantic html tags!
+
+## Submission
+* At the end of the lab, drop a link to your forked repl in the slack thread your instructor will start!
+
+## Comp Images
```

Comment:
Do you think they know enough layout tools to make something sort of like this. I can imagine a ton of questions about how to get this next to this, and how to get these to stack. And I would use grid or flexbox, so I'm not sure how they would approach trying to get layouts?

Reply:
I do not think they will know enough 😆 I'll add a note that we do not expect that they will finish, but to try to get as much of the style elements as possible.  We could have them do a little flex-box froggy to start the lab so that they know it's _possible_.  But realistically, this will be an exercise in patience... and hopefully give them some developer empathy for our designer friends.